examples/howto/marginalizing-models.ipynb

```diff
@@ -0,0 +1,1440 @@
+{
```

Comment:
<p>Since both models are mixture, maybe call this <code>prebuilt_mixture</code> and the other <code>custom_mixture</code> or something along those lines. I would also call <code>az.summary</code> in the same cell where fit is done, because reading <code>idata</code> doesn't immediately convey which model it came from (I usually scroll up and down when reading examples, so it can be a bit confusing)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/628/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>